cointop/cointop.go

```diff
@@ -284,9 +288,12 @@ func NewCointop(config *Config) (*Cointop, error) {
 			refreshRate:           60 * time.Second,
 			selectedChartRange:    DefaultChartRange,
 			shortcutKeys:          DefaultShortcuts(),
-			sortBy:                DefaultSortBy,
-			page:                  0,
-			perPage:               int(perPage),
+			selectedView:          CoinsView,
+			viewSorts: map[string]*sortConstraint{
+				CoinsView: {DefaultSortBy, false},
+			},
+			page:    0,
+			perPage: int(perPage),
```

Comment:
what's going on with the indent here?

Reply:
That's the normal behavior of lint tool, I can't answer why 😂